src/Processors/QtiV2/Out/AssessmentItemBuilder.php

```diff
@@ -47,12 +45,8 @@ public function build($itemIdentifier, $itemLabel, array $questions, $content =
         $responseProcessingTemplates = [];
         foreach ($questions as $question) {
             $questionData = $question->to_array();
-            if (isset($questionData['data']['validation']['valid_response']['score'])) {
-                $score = $questionData['data']['validation']['valid_response']['score'];
-                $assessmentItem->setOutcomeDeclarations($this->buildOutcomeDeclarations($score));
-            } else {
-                $assessmentItem->setOutcomeDeclarations($this->buildOutcomeDeclarations(0));
-            }
+            $assessmentItem->setOutcomeDeclarations($this->buildOutcomeDeclarations(0));
```

Comment:
As param has already been remove from buildOutcomeDeclarations, is there any reason to keep 0 as param?

Reply:
Its look like I forgot to remove this 0. Removed now.